# ros_control_boilerplate Settings
# Settings for ros_control control loop
generic_hw_control_loop:
    loop_hz: 100
    cycle_time_error_threshold: 0.02

# Settings for ros_control hardware interface
hardware_interface:
    joints:
        - left_drive_wheel_joint
        - right_drive_wheel_joint
    sim_control_mode: 1 # 0: position, 1: velocity
    
jaguar_hw_interface:
    left_id: 2
    right_id: 4
    encoder_ticks: 360
    enable_brake: true

    left_ff: 0.00248960589 # 1.0/401.67
    right_ff: 0.00248960589 # 1.0/401.67

    # Not currently used
    odom_rate_ms: 20
    diag_rate_ms: 500

    left_pid:
        p: 0.0
        i: 0.0
        d: 0.0
        i_clamp: 0.0

    right_pid: 
        p: 0.0
        i: 0.0
        d: 0.0
        i_clamp: 0.0


